packages/toast/service.js

```diff
@@ -15,6 +15,7 @@ const initArea = () => {
 const insertToArea = (area, el, id) => {
   el.setAttribute('id', `zi-toast-${id}`)
   area.appendChild(el)
+  setTimeout(() => el.classList.add('visible'), 0)
```

Comment:
clear timer by manually. (this is a convention, not a pref optimization)
```suggestion
  const timer = setTimeout(() => {
    el.classList.add('visible')
    ...
  }, 0)
```

Reply:
👌